pkg/integrations/v4/dm/emitter_test.go

```diff
@@ -250,7 +250,6 @@ func TestEmitter_Send_failedToSubmitMetrics_dropAndLog(t *testing.T) {
 	ctx.SendDataWg.Wait()
 
 	entry := hook.LastEntry()
```

Comment:
Since `require.NotEmpty(t, hook.AllEntries())` was tended to check that there were logs in the hook could you add
a nil checker:
`require.NotNil(t, entry)`